en/docs/administer/browsing-the-h2-database.md

```diff
@@ -62,7 +58,7 @@ browse through it.
       </tr>
       <tr class="even">
       <td><strong>JDBC URL</strong></td>
-      <td>location of the H2 database</td>
+      <td>location of the H2 database. All the default databases are located at `<IS_HOME>/repository/database` location.</IS_HOME></td>
       </tr>
```

Comment:
This should be "All the default databases are located at `<IS_HOME>/repository/database`." We can remove the "location.<IS_HOME>" part.

Reply:
fixed